src/resources/js/events-admin.js

```diff
@@ -258,10 +254,10 @@ jQuery( document ).ready( function( $ ) {
 	 */
 	var $date_format      = $( '[data-datepicker_format]' ),
 		$view_select      = $( '.tribe-field-dropdown_select2 select' ),
-		viewCalLinkHTML   = $( document.getElementById( 'view-calendar-link-div' ) ).html(),
+		viewCalLinkHTML   = $( '#view-calendar-link-div' ).html(),
```

Comment:
Personally, I like this style because it's far more readable, however it's arguably less performant and other reviews may ask you to change it back